mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -318,8 +318,11 @@ class CompletionsProvider(
           filterText,
           Some(completionPos.copy(start = start).toEditRange),
         )
-      case CompletionValue.NamedArg(label, _) =>
-        mkItem(ident, ident.replace("$", "$$")) // escape $ for snippet
+      case CompletionValue.NamedArg(label, _, insertText) =>
+        mkItem(
+          ident,
+          insertText.getOrElse(ident.replace("$", "$$")),
```

Comment:
If we remove insertText and use label instead, should we still replace "$" with "$$" here, because it might not be needed when we have completion with default argument. Can we do this `replace` in `NamedArgCompletions`?